### PR TITLE
[14.0][MIG] hr_expense_tier_validation

### PR DESCRIPTION
**Pull request by aphon61bank**
add _tier_validation_manual_config = False .

### CODE REVIEW

**Review by dsolanki-initos — APPROVED** [review]
Code review and Functional test done. LGTM

File: hr_expense_tier_validation/views/hr_expense_sheet_view.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            </header>
+            <xpath expr="//form/div[hasclass('oe_chatter')]" position="before">
+                <field
+                    name="review_ids"
+                    widget="tier_validation"
+                    attrs="{'invisible':[('review_ids', '=', [])]}"
+                />
+            </xpath>
+        </field>
+    </record>
```
Comment (kittiu) [inline hr_expense_tier_validation/views/hr_expense_sheet_view.xml]: No need this part as _tier_validation_manual_config = False
```suggestion
```

**Comment by dsolanki-initos** [comment]
@ps-tubtim can you please squash the commits and ask for re-review so that PR can be merged soon

**Comment by ps-tubtim** [comment]
@dsolanki-initos Thank you. I squashed the commits.

**Comment by dsolanki-initos** [comment]
@kittiu can you please merge this PR?

**Review by kittiu — APPROVED** [review]

**Review by Saran440 — APPROVED** [review]
Functional Test :+1: 

Note:
In core odoo14.

**Case: Create Expense to report**
it will change state report to "Submitted". So this report skipped tier validation and you can approve this report.

This PR add state tier vaidation from submit > approve

**Pull request merged**